build/android/dependency

```diff
@@ -12,23 +12,18 @@ ANDROIDDIR=$BUILDDIR/android
 
 NDK_DIR=$1
 SPEC=$2
-API=21
-if [ "$3" != "" ]; then
-    API=$3
-fi
 
 echo ""
 echo "Cross-compiling $SPEC for all archs"
 
-ALL_ARCHS="aarch64-linux-android arm-linux-androideabi arm-linux-androideabi-v7a mipsel-linux-android mips64el-linux-android x86 x86_64"
+ALL_ARCHS="armeabi armeabi-v7a arm64 mips mips64 x86 x86_64"
```

Comment:
The script to generate the toolchain has changed (the bash one has been deprecated for quite some time in favor of the python one that we're using now); architecture names changed accordingly